content/docs/instrumenting/exporters.md

```diff
@@ -226,6 +226,7 @@ separate exporters are needed:
    * [Traefik](https://github.com/containous/traefik)
    * [VerneMQ](https://github.com/vernemq/vernemq)
    * [Weave Flux](https://github.com/weaveworks/flux)
+   * [Apache Zipkin (incubating)](https://zipkin.incubator.apache.org/)
```

Comment:
I put this at the end because Zipkin would be last

Reply:
There's no need to mention that it's incubating, and Apache would go first. I always hear it referred to as just Zipkin, and that's what's on its webpage.

I gather this isn't on /metrics, can you link to the docs saying where it is?